build/build.go

```diff
@@ -0,0 +1,12 @@
+/*
+Package build contains build specific information.
+*/
```

Comment:
can you do `//` instead, `/* */` aren't usually used in go and definitely not in this code base.

Reply:
I think I got carried away with `/*` for cmds where you typically want formatted docs for examples etc. Thanks.

Reply:
Resolved